publication-commons/src/main/java/no/unit/nva/publication/service/impl/TicketService.java

```diff
@@ -49,6 +51,7 @@ protected TicketService(AmazonDynamoDB client,
                             Supplier<SortableIdentifier> identifierProvider) {
         super(client);
         this.identifierProvider = identifierProvider;
+        tableName = RESOURCES_TABLE_NAME;
```

Comment:
Why not reference constant directly?

Reply:
To send not default table name to this method: fetchByIdentifier
 which is used in all database services to access resources from database